desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -12,6 +12,9 @@
 #include "core/display.h"
 #include "core/divelist.h"
 
+#include <stdlib.h>
+#include <stdio.h>
+
```

Comment:
why do we need these two headers? Did you have some `fprintf()` in there for testing and kept the `#include` lines? Nothing in the code you added should need these headers...